Dockerfile

```diff
@@ -12,40 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+## Note - This file is not used in the current build and release process
 
 # syntax=docker/dockerfile:1
 
 ############################# BUILD ###############################
-# This stage builds the HarbourBridge binary using its source code.
+# This stage builds the Spanner migration tool binary using its source code.
 
 # Add golang as the base image for the build stage.
 FROM golang:1.18.3 AS build
 
-# Set the default work directory as “/harbourbridge_bin” in the container.
-WORKDIR /harbourbridge_bin
+# Set the default work directory as “/spanner_migration_tool_bin” in the container.
+WORKDIR /spanner_migration_tool_bin
 
-# Copy HarbourBridge’s source code into the working directory.
+# Copy Spanner migration tool’s source code into the working directory.
 COPY . ./
-# Download HarbourBridge’s dependencies mentioned in the go.mod file.
+# Download Spanner migration tool’s dependencies mentioned in the go.mod file.
 RUN go mod tidy 
-# Compile the source code and store the binary in the work directory /harbourbridge_bin.
+# Compile the source code and store the binary in the work directory /spanner_migration_tool_bin.
 RUN make build-static  
 
 ########################### RELEASE ###############################
-# This stage consists of the GCloud SDK and the HarbourBridge binary.
+# This stage consists of the GCloud SDK and the Spanner migration tool binary.
 
 # Add the Ubuntu base image from the Google Container Registry.
 FROM gcr.io/cloud-marketplace/google/ubuntu1804@sha256:d9fe979ab72ff02f5b5d95c59cda6681aa73039fbf9b46596935267cf26c692e
 
 # Set the default working directory.
-# This directory binds with the host directory that is to store the output files generated by HarbourBridge.
-WORKDIR /harbourbridge/spanner_migration_tool_output
-# Create directory to store the HarbourBridge binary. 
-RUN mkdir /harbourbridge/bin
+# This directory binds with the host directory that is to store the output files generated by Spanner migration tool.
+WORKDIR /spanner_migration_tool/spanner_migration_tool_output
+# Create directory to store the Spanner migration tool binary. 
+RUN mkdir /spanner_migration_tool/bin
 # Create directory to store the dump files. This directory binds with the host directory that contains the dump files of the source database.
-RUN mkdir /harbourbridge/source_dump
-# Copy the HarbourBridge binary from the build stage's harbourbridge_bin directory to the release stage's harbourbridge/bin directory.
-COPY --from=build /harbourbridge_bin/harbourbridge /harbourbridge/bin
+RUN mkdir /spanner_migration_tool/source_dump
+# Copy the spanner-migration-tool binary from the build stage's spanner_migraiton_tool_bin directory to the release stage's spanner_migration_tool/bin directory.
```

Comment:
nit spell check: spanner_migraiton_tool_bin  to spanner_migration_tool_bin

Reply:
Done.